projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/protect-value-section.tsx

```diff
@@ -0,0 +1,113 @@
+import { Gridicon } from '@automattic/jetpack-components';
+import { Popover } from '@wordpress/components';
+import { useViewportMatch } from '@wordpress/compose';
+import { __, sprintf } from '@wordpress/i18n';
+import { useState } from 'react';
+import { PRODUCT_STATUSES } from '../../../constants';
+import useProduct from '../../../data/products/use-product';
+import { getMyJetpackWindowInitialState } from '../../../data/utils/get-my-jetpack-window-state';
+import { timeSince } from '../../../utils/time-since';
+
+import './style.scss';
+
+const ProtectValueSection: React.FunctionComponent = () => {
+	const slug = 'protect';
+	const { detail } = useProduct( slug );
+	const { status } = detail;
+	const isProtectActive =
+		status === PRODUCT_STATUSES.ACTIVE || status === PRODUCT_STATUSES.CAN_UPGRADE;
+
+	return isProtectActive ? <WithProtectValueSection /> : <NoProtectValueSection />;
+};
+
+export default ProtectValueSection;
+
+const WithProtectValueSection: React.FunctionComponent = () => {
```

Comment:
Might want to import FC directly. Also it may not be necessary in this case since TypeScript should detect the correct type here automatically 🤔. Personally I like using FC when there are props because it looks nice in the code 😅 but there was a P2 written recently that suggests FC by itself may not be useful: pdWQjU-Np-p2

Reply:
Yeah, I'll use FC where there are props in use. 👍